apiary.apib

```diff
@@ -2474,6 +2474,15 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + hourly_rate (object, required)
+            + One Of
+                + Properties
+                    + type: `user` (string, required)
+                + Properties
+                    + type: `task` (string, required)
```

Comment:
B : `task` is something that doesn't exist anymore... shouldn't it be `work_type`?